pypeit/pypeit.py

```diff
@@ -275,106 +268,51 @@ def get_std_outfile(self, standard_frames):
             msgs.error('Could not find standard file: {0}'.format(std_outfile))
         return std_outfile
 
-    def calib_all(self, run=True):
```

Comment:
are you sure the `run` option isn't used?

Reply:
It was only used for the `parse_calib_id` script, which has been deprecated.